src/core/server/stacks/createComment.ts

```diff
@@ -175,6 +176,7 @@ export default async function create(
   );
 
   await updateUserLastCommentID(redis, tenant, author, comment.id);
+  await updateStoryLastCommentedAt(mongo, tenant, story, now);
```

Comment:
Because the ordering of these is not important, you can optimize this a bit with:

```ts
await Promise.all([
  updateUserLastCommentID(redis, tenant, author, comment.id),
  updateStoryLastCommentedAt(mongo, tenant, story, now),
]);
```